spec/models/chesspiece_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe ChessPiece do
+  describe "#horizontal_move?" do
+     context "given a piece at [4, 4]" do
+       let(:ChessPiece) {ChessPiece.create(position_x: 4, position_y: 4)}
+       it "will return true when trying to move to [5,4]" do
+        expect(ChessPiece.horizontal_move?(4,4)).to be_truthy
```

Comment:
`horizontal_move?` is an `instance` method. Here you're using it as a `Class` method.